app/src/main/java/org/p2p/wallet/utils/SolanajTimberLogger.kt

```diff
@@ -0,0 +1,20 @@
+package org.p2p.wallet.utils
+
+import org.p2p.solanaj.utils.SolanjLogger
+import timber.log.Timber
+
+private const val SOLANAJ_TAG = "Solanaj module"
```

Comment:
```suggestion
private const val SOLANAJ_TAG = "SOLANAJ_MODULE"
```
I think this format will easier to find in logcat